sdk/trace/trace_test.go

```diff
@@ -1045,7 +1045,7 @@ func TestEndSpanTwice(t *testing.T) {
 	}
 
 	ro := span.(ReadOnlySpan)
-	if ro.EndTime() != et1 {
+	if !ro.EndTime().Equal(et1) {
```

Comment:
`et1` is monotonic but EndTime returns a non-monotonic timestamp since we create it via time.Unix.